src/lib/swapper/swappers/OsmosisSwapper/utils/helpers.ts

```diff
@@ -180,15 +179,14 @@ const getPoolRateInfo = (
   const buyAmountCryptoBaseUnit = buyAssetInitialPoolSize.minus(buyAssetFinalPoolSize).toString()
   const rate = bnOrZero(buyAmountCryptoBaseUnit).dividedBy(sellAmount).toString()
   const priceImpact = bn(1).minus(initialMarketPrice.dividedBy(finalMarketPrice)).abs().toString()
-  const tradeFeeBase = bnOrZero(buyAmountCryptoBaseUnit).times(bnOrZero(pool.pool_params.swap_fee))
-  const buyAssetTradeFeeUsd = tradeFeeBase
-    .dividedBy(bn(10).exponentiatedBy(OSMOSIS_PRECISION))
+  const buyAssetTradeFeeCyptoBaseUnit = bnOrZero(buyAmountCryptoBaseUnit)
+    .times(bnOrZero(pool.pool_params.swap_fee))
```

Comment:
Sanity checked the previous logic was also correct, and indeed it is
https://github.com/osmosis-labs/telescope/blob/2db2d4e07fa8ebcac759c2ffc88f1303e598e543/__fixtures__/osmojs/osmosis/proto/osmosis/concentrated-liquidity/pool.proto#LL57C1-L57C1
>   // swap_fee is the ratio that is charged on the amount of token in.